runtime/gc_api/HeapIteratorAPI.cpp

```diff
@@ -563,6 +569,7 @@ j9mm_iterate_all_continuation_objects(J9VMThread *vmThread, J9PortLibrary *portL
 		}
 		continuationObjectList = continuationObjectList->getNextList();
 	}
+	j9tty_printf(PORTLIB, "j9mm_iterate_all_continuation_objects count=%zu, time=%zu ms\n", count, j9time_hires_delta(startTime, j9time_hires_clock(), J9PORT_TIME_DELTA_IN_MICROSECONDS));
```

Comment:
I guess this is temporary debugging code. Would it be useful to convert it to something permanent (like tracepoint)?

Reply:
looks all GC trace points are defined in omr/gc/base/j9mm.tdf, should we add this one to omr code?